src/components/ViewErrorFallback.tsx

```diff
@@ -34,7 +34,8 @@ const Title = styled(Typography)`
   line-height: 1.25;
 `
 
-const ErrorFallback: React.FC<FallbackProps> = ({ error, resetErrorBoundary }) => {
+const ErrorFallback: FallbackRender = ({ error, componentStack, resetError }) => {
+  console.error(`An error occured in ${componentStack}`)
```

Comment:
I guess every `error` logged will be a separate event in Sentry. Maybe it'd make sense to join both of those so we always know which stack links to each error? Something like this?
```js
console.error('An error occurred', {stack: componentStack, error}')
```

Reply:
Also `occured` -> `occurred`

Reply:
Thanks, changed that